homeworks/nazar_kostiv/HW02/job.py

```diff
@@ -0,0 +1,37 @@
+"""Homework #2 by Nazar K.
+
+Task:
+There are two numbers. Each of them can be negative, floating etc.
+You need to create a job which will accept two numbers as the arguments.
+This job has to run in parallel 4 different tests: multiplication, division, addition, subtraction.
+Each of these tests will perform an appropriate math operation on given numbers.
+A test has to fail if the calculation result is less then 0, otherwise, the test is passed."""
+
+# pylint: disable=C0103
+import argparse
+import sys
+from pyats.easypy import Task
+
+
+def new_task(runtime, test, num1, num2):
+    return Task(testscript=test, runtime=runtime, num1=num1, num2=num2)
+
+
+def main(runtime):
+    tasks_path = ['add.py', 'mul.py', 'sub.py', 'div.py']
+    args, sys.argv[1:] = parser.parse_known_args(sys.argv[1:])
+    tasks = [new_task(runtime, path, args.num1, args.num2) for path in tasks_path]
+    for task in tasks:
+        task.start()
+    for task in tasks:
+        if task.is_alive():
+            task.wait()
+
+
+parameters = {
```

Comment:
@ProffesorRabbit Why do you need this dictionary?

Reply:
@extsoft I decided to store there default values for num1 and num2 if this two weren't entered in command.

Reply:
@ProffesorRabbit Okay, but why do we need a dictionary? We can just set them without it.

Reply:
@extsoft i thought it will be looked like 'magic' numbers

Reply:
It won't be as the like has a context about these numbers.

Reply:
@extsoft OK, removed dict and setted those numbers as default parameters